files/lib/event/interaction/bulk/user/ConversationBulkInteractionCollecting.class.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace wcf\event\interaction\bulk\user;
+
+use wcf\event\IPsr14Event;
+use wcf\system\interaction\bulk\user\ConversationBulkInteractions;
+
+/**
+ * Indicates that the provider for conversation bulk interactions is collecting interactions.
+ *
+ * @author Olaf Braun
+ * @copyright 2001-2025 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since 6.2
+ */
+final class ConversationBulkInteractionCollecting implements IPsr14Event
+{
+    public function __construct(public readonly ConversationBulkInteractions $param)
```

Comment:
```suggestion
    public function __construct(public readonly ConversationBulkInteractions $provider)
```
Please avoid generic variables names like `$param`.